chexus/_impl/models.py

```diff
@@ -29,11 +29,19 @@ class BucketItem(object):
             provided.
     """
 
-    def __init__(self, file_path, file_name=None, checksum=None, key=None):
+    def __init__(
+        self,
+        file_path,
+        file_name=None,
+        checksum=None,
+        key=None,
+        extra_args=None,
```

Comment:
Should probably explicitly call this "headers" since we're assigning it to self.headers